pysph/examples/dam_break_2d.py

```diff
@@ -67,19 +67,37 @@ def add_user_options(self, group):
             '--staggered-grid', action="store_true", dest='staggered_grid',
             default=False, help="Use a staggered grid for particles.",
         )
+        group.add_argument(
+            '--interp-method', action="store", type=str, dest='interp_method',
+            default='shepard', help="Specify the interpolation method.",
+        )
+        group.add_argument(
```

Comment:
Same here.  Please make this a choice so the possible values are also documented.